css/style.css

```diff
@@ -103,20 +102,56 @@ p {
   padding-bottom: 100px;
 }
 
-.second-section-columns img {
+.information-section-columns img {
   width: 100px;
 }
 
+.quote-section {
+  display: flex;
+}
+
+.quote-section img {
+  width: 100%;
+}
+
+.quote-rectangle {
+  background-color: E0633A;
+  border: 5px solid blueviolet;
+}
+
+.quote-text {
+  padding: 20px 40px 0px;
+  font-style: italic;
+}
```

Comment:
Great, I see CSS classes are better named and I guess you are writing some CSS class for store.html as well. Just remember to split them to separate CSS files to improve CSS coverage (i.e. improve better code utilisation)